src/main/java/org/cryptacular/KeyLookup.java

```diff
@@ -0,0 +1,24 @@
+/* See LICENSE for licensing and NOTICE for copyright. */
+package org.cryptacular;
+
+import javax.crypto.SecretKey;
+
+/**
+ * Interface to allow custom implementations of secret key lookups.
+ *
+ * @deprecated  In newer versions, KeyLookup is replaced by java.util.Function&lt;String, SecretKey&gt; instances.
+ *
+ * @author  Middleware Services
```

Comment:
You can credit yourself/your org here if you want.

Reply:
Thanks, but given that this was mostly derivative work, I'm completely fine just using Middleware Services here.